### ISSUE TITLE
surveymyopinion.researchnow.com

### ISSUE THREAD

**Issue by Eow2022yia** [AdguardTeam/AdGuardSDNSFilter#1138]
surveymyopinion.researchnow.com is blocked on AdGuard DNS,
This is a survey portal that is only accessible via direct invitation via a market research company, it is not an ad or tracker, please whitelist, thank you.

**Comment by Eow2022yia** [comment]
> sounds like data collection not required for user needs. Market research aka data hoarding. 

Hmmm, you seem to be making an assumption by the domain title, probably why it's blocked in the first place to be fair.

If you don't participate in voluntary market research and surveys for paid incentives you won't understand by default.

It is only accessible by survey websites like Swagbucks etc, it is not featured anywhere else, therefore it is not a tracker, yes it does collect data but only when a user consents to this when signing up for the survey panel in the first place, hence why when you click on the domain itself it presents a white blank page (when not blocked of course).

This being blacklisted is breaking the functionality and I have to change my DNS every time simply to access it.

This is not a tracker or advertisement and the idea of DNS-based is not to break websites or/and portals.

Thanks.

**Comment by Eow2022yia** [comment]
> Users should be protected against this crap. They have no idea what they "voluntarily" are giving away. 

Protected? As I said, you obviously have no idea how survey sites work, I mean the best intentions when I say that maybe you should do some research on it yourself.

AdGuard DNS is for ad, tracking and malware blocking, not for hardcore content blocking and breaking websites, services or programs you don't personally agree with, you should create a separate list for that.

**Comment by Eow2022yia** [comment]
> Well I present you: https://github.com/ph00lt0/blocklist
> 
> 
> 
> I fundamentally disagree with your view. DNS blocklists are there to protect users from malware as you said. Data aggregation of users who have no idea what happens with this should be considered malware. Users unfortunately are not able to make the right decisions when it comes to sharing their data. If people actually would understand the implications they would never want to participate in this. This is the general problem of how companies like the ones you are defending are abusing people who lack the understanding of their rights and how they are being manipulated.  Maybe instead of selling your soul you should do some _research_ :'D. It seems you _obviously have no idea how survey sites work_.

People can make their own choices and can do their own research regarding it if they wish too, I'm pretty sure they are aware of links such as these when they sign up to the survey panel and input their personal information hence why these sites have privacy policies etc. Please also remember  this link in-question is only accessible via invite-only.

Although I respect your controversial opinion regarding this, what you are saying makes zero sense, is facebook.com and google.com blocked on your "blocklist" too? They track, sell and share your data too?

I'm curious on what your limits are on this?

**Comment by Eow2022yia** [comment]
> @Eow2022yia you make so many assumptions, really stop it. I really would like to know the percentage of your users that actually reads the privacy policy. That besides the amount of people who actually understand what it you do with the data usually is zero.
> 
> 
> 
> I gladly address your concern on the decisions we made for our blocklist. 
> 
> Facebook and Google are not blocked although their tracking scripts are. They surely track people. However these are services people require for their day to day activities. Surveys surely are not part of this, as they don't stop the user from looking something up, being in touch etc. The by you reported domain name service no purpose for the end user. 

Hi,

No assumptions made here, I am reading your statements and responding accordingly.

Companies have to provide correct and accurate privacy policies otherwise they can land themselves into legal issues with local data protections laws.

In regards to your statement, I think you'll find many people use survey websites to earn cash in there spare time.

I respect your opinion but this is adblocking, not a firewall for services you don't like nor agree with.

However you can obviously do as you wish with your own blocklists but please remember it won't be popular with people if it breaks services, regardless of your opinion of it.

You should take a look into OISD's list, it block the ads, trackers and malware, not the functionality or services.

Thank you :)
